lib/elixir_bench/github/client_http.ex

```diff
@@ -0,0 +1,50 @@
+defmodule ElixirBench.Github.ClientHTTP do
+  @moduledoc """
+  The Github HTTP client.
+
+  This client uses HTTP requests to gather data from yml files in given
+  repository hosted on Github. It uses the `:hackney` library to actually send
+  the requests and the `yamerl` library to decode the data in the yml format.
```

Comment:
mentioning libraries used under the hood is too application specific imo - we can do without that. Those might change or not - whoever uses the module doesn't need to know about them though :)